quinn-h3/src/qpack/encoder.rs

```diff
@@ -145,6 +145,9 @@ pub fn on_decoder_recv<R: Buf>(table: &mut DynamicTable, read: &mut R) -> Result
     while let Some(instruction) = parse_instruction(read)? {
         match instruction {
             Instruction::Untrack(stream_id) => table.untrack_block(stream_id)?,
+            Instruction::UntrackInfaillible(stream_id) => {
```

Comment:
```suggestion
            Instruction::UntrackInfallible(stream_id) => {
```

This is probably a gallicism, you made a similar mistake in some of your earlier commit messages. :)

(BTW, the commit message also has a typo, `streaam` -> `stream`.)

The vocabulary "untrack infallible" seems a little hard to understand to me. Is this what the spec calls it?

Reply:
This is not really defined with a specific terminology in the draft. It only states that stream cancellation instruction on unknown or already deleted stream-related reference block can be ignored. 

Now I figure out this should have another behaviour than `Untrack` nevertheless, at it only removes one block of reference for the stream (there are one for headers and trailers respectively). I'll just rename this to `StreamCancel` or something similar.